scripts/defaults

```diff
@@ -2,7 +2,9 @@ echo "this is the file defaults"
 
 export PYTHON=${PYTHON:-"python"}
 export VIRTUAL_ENV=${VIRTUAL_ENV:-"$PWD/socorro-virtualenv"}
-export BUILD_TYPE=${BUILD_TYPE:-"tar"}
+# Valid values for BUILD_TYPE are "rpm" (which we use in socorro-infra) and
+# "ci" # which we use in Travis CI.
+export BUILD_TYPE=${BUILD_TYPE:-"ci"}
```

Comment:
When Travis runs, the value is "tar". When our deploys run, the value is "rpm". We don't do tar or deb packaging anymore. So I changed use of `BUILD_TYPE` in the scripts to do things based on two values: "rpm" and "ci".